src/main/markdown/doc/latest/polymer-tutorial/create.md

```diff
@@ -33,12 +33,14 @@ For the **TodoList** project, we'll need to run `webAppCreator` with the followi
 
       _**Tip**: If you include the GWT SDK folder in your PATH environment variable, you won't have to specify the full path._
 
+      You may have to modify the `pom.xml` before you can run the application. Add `<type>pom</type>` to the gwt dependency, otherwise you will encounter an error. 
 
 2. Run the application in [SuperDevMode](articles/superdevmode.html).
 
      To check that the project was created correctly start the new app in SuperDevMode.
 
         $ cd TodoListApp
+        $ mvn war:exploded
```

Comment:
good point, although the user only needs to run `mvn war:exploded` _once_ unless he runs `mvn clean` at some point, so I think it's safe. Alternatively, providing them with a `pom.xml` that just includes the `maven-war-plugin` (which is what `mvn war:exploded` calls) like you do [in the gwt-polymer-elements demo](https://github.com/vaadin/gwt-polymer-elements/blob/master/demo/pom.xml) would be a way to make absolutely sure the resources get copied.